hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,5 @@
+RELEASE_TYPE: patch
+
+This release fixes the type hint on `register_type_strategy`. The second
```

Comment:
This is a great patch note, thanks!  Two nitpicks though: one of the ways that ReStructured text diverges from markdown is that single backticks are used in various cross-reference constructs, so you need double-backticks to do literal formatting.  The other is that we want to make a link for `register_type_strategy`, which looks like this:

    :func:`~hypothesis.strategies.register_type_strategy`

That is: type of reference in colons, full target in backticks, and the `~` indicates that we only want the local name in the text.  [Docs here[(http://www.sphinx-doc.org/en/stable/markup/inline.html) for Sphinx markup :smile: